File: 2018-2/ap1/tabuada.cpp
#include<stdio.h>	
#include<conio.h>
main()

{int x,tab,s=0;

	printf ("\nDigite um numero para saber seus multiplos:");
	scanf ("%i",&tab);
	
	for (x=1;x<=10;x++)
{
	printf ("\n%i X %i = %i",tab,x,tab*x);
}

getche();

}

File: 2019-2/Ap3/tabuada.java
import javax.swing.*;

public class tabuada {
    public static void main(String[] args) {
        int i;
        int cont=0;
        int n;
        int result;
        String teclado;

        teclado = JOptionPane.showInputDialog("Digite um numero para saber seus multiplos:");
        n = Integer.parseInt(teclado);
        for (i=0; i<=10; i++){
            result = n * i;
            JOptionPane.showMessageDialog(null,n +" x "+ cont++ +" = " + result);
        }
    }
}
